openmetadata-ui/src/main/resources/ui/src/styles/tailwind.css

```diff
@@ -301,20 +301,6 @@
     @apply tw-bg-primary tw-translate-x-4;
   }
 
-  body .content-container h1,
-  body .content-container h2 {
-    @apply tw-text-h3;
-  }
-  body .editor-wrapper ul,
-  body #description ul,
-  body .content-container ul {
-    @apply tw-list-disc;
-  }
-
-  body .content-container p {
-    word-break: break-word;
-  }
-
```

Comment:
Any reason we are removing it?

Reply:
It was for old markdown parsers and now it’s not being used, so removed it.